core/drivers/crypto/caam/utils/utils_mem.c

```diff
@@ -413,6 +413,9 @@ enum caam_status caam_cpy_block_src(struct caamblock *block,
 	enum caam_status ret = CAAM_FAILURE;
 	size_t cpy_size = 0;
 
+	if (!src->data)
+		goto end_cpy;
```

Comment:
Why not return directly instead? There's nothing special done at the label.

Reply:
Ok